vault/logical_system_test.go

```diff
@@ -2420,6 +2420,62 @@ func TestSystemBackend_enableAudit(t *testing.T) {
 	}
 }
 
+func missingPayloadTest(path string, data map[string]interface{}) func(t *testing.T) {
+	return func(t *testing.T) {
+		_, b, _ := testCoreSystemBackend(t)
+		req := logical.TestRequest(t, logical.UpdateOperation, path)
+		req.Data = data
+		resp, err := b.HandleRequest(namespace.RootContext(nil), req)
+		if err == nil {
+			t.Fatalf("err: %v", err)
```

Comment:
We know err is nil here, so maybe a simple literal string like "missing payload" makes more sense as the fatal output than "err: nil".

Reply:
OK, I decided to return a message explaining the error.